deploy-ocs/deploy.sh

```diff
@@ -55,7 +55,37 @@ if ! ./verify.sh; then
         ALLSPOKES=$(yq e '(.spokes[] | keys)[]' ${SPOKES_FILE})
     fi
 
+    index=0
     for spoke in ${ALLSPOKES}; do
+        echo ">>>> Nuking storage disks for: ${spoke}"
```

Comment:
The validation for `root_disk` parameter on the spoke.yaml file it's out of this PR? is that right? 

if that it's the case, I think it's necessary to merge both together in order to avoid the deletion of the OS Disk by accident

Reply:
Yes, it's in the other one, so that it could be different